job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/step/StepContext.groovy

```diff
@@ -723,6 +723,42 @@ class StepContext extends AbstractExtensibleContext {
     }
 
     /**
+     * @since 1.36
+     */
+    @RequiresPlugin(id = 'shiningpanda')
+    void python(@DslContext(PythonContext) Closure closure) {
+        PythonContext context = new PythonContext()
+        ContextHelper.executeInContext(closure, context)
+
+        stepNodes << new NodeBuilder().'jenkins.plugins.shiningpanda.builders.PythonBuilder' (plugin: 'shiningpanda') {
```

Comment:
The plugin attribute `(plugin: 'shiningpanda')`, the DSL does not generate the plugin attributes for any plugin.